internal/commands/reminder.go

```diff
@@ -23,19 +23,33 @@ func requestStatus(ctx context.Context, client bot.Client, logger log.Logger, re
 		if err != nil {
 			logger.Error(
 				ctx,
-				"command/reminder.requestStatus GetIncident error",
-				log.NewValue("channelID", incident.ChannelId),
+				log.Trace(),
```

Comment:
We should add this to every log, very good!

Reply:
Thank you :)